core/src/main/java/org/keycloak/representations/AccessToken.java

```diff
@@ -90,13 +90,24 @@ public static class Authorization implements Serializable {
         @JsonProperty("permissions")
         private Collection<Permission> permissions;
 
+        @JsonProperty("aud")
+        private String audience;
+
```

Comment:
`aud` is already available in JsonWebToken here, and can be either a string or a string array:
https://github.com/keycloak/keycloak/blob/ceeb604f1cf73db5dbd70a3b9e4ec56606a1982b/core/src/main/java/org/keycloak/representations/JsonWebToken.java#L55-L58